workers/unity/Packages/com.improbable.gdk.core/Components/SpatialEntityId.cs

```diff
@@ -3,6 +3,9 @@
 
 namespace Improbable.Gdk.Core
 {
+    /// <summary>
+    ///     ECS component which contains the Spatial Entity ID.
```

Comment:
are we saying Spatial or SpatialOS?

Reply:
Probably should be `SpatialOS` - I think that's the notation I've gone with in most places